todo/src/containers/container.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react";
+import { connect } from 'react-redux';
+import { addTodo, removeTodo, toggleTodo } from '../actions/todoActions';
+
+class Todo extends Component {
+    state = {
+        todos: '',
+        toggle: false
+    }
+
+updateTodo = (event) => {
+    console.log(event.target);
+    this.setState({todos: event.target.value});
+}
+
+addTodoInput = (event) => {
+   const {todos, toggle} = this.state
+    console.log(event.target);
+    event.preventDefault();
+    this.props.addTodo({todos, toggle});
+}
+
+removeTodoInput = (event) => {
+    this.props.removeTodo(event.target.getAttribute('id'));
+}
+
+toggleTodoInput = (event) => {
+    this.props.toggleTodo(event.target.getAttribute('id'));
+}
+
+render() {
+    console.log(this.props);
+    return (
+        <div className="Todo">
+            <form onSubmit={this.addTodoInput}>
+            <input
+                onChange={this.updateTodo}
+                placeholder="todo"
+                value={this.state.todos}
+            />
+            <button value="addTodo" onClick={this.addTodoInput} type="submit">Add to todolist</button>
+            </form>
+            <ul>
+                {this.props.todos.map((todo, index) => {
+                    console.log(todo.toggle);
+                    return(
+                        <li key={index} id={index} onClick={this.toggleTodoInput}>{todo.todos}</li>
```

Comment:
Since we added `id` to our `todo` object, we can now access it here, as well as the `toggle` state:

```
<li key={index} id={todo.id} onClick={this.toggleTodoInput} className={todo.toggle ? 'line-through' : null}>{todo.todos}</li>
```

Reply:
`todo.toggle` is a Boolean, so this will assign a class name whether its true or false. You'll have to define what `'line-through` looks like in your CSS file.

Reply:
I just remember you can do this:

`style={{ textDecoration: todo.toggle ? 'line-through' : 'none' }}`

instead of using `className`